File: src/codeforces/accepted/133A.cc
// Problem #    : 133A
// Created on   : 2018-10-14 09:58:00
#include <bits/stdc++.h>
#define FR(i, n) for (int i = 0; i < (n); ++i)
using namespace std;

typedef long long ll;
typedef pair<int, int> ii; 
typedef vector<int> vi;    

int main() {

   ios_base::sync_with_stdio(false);
   cin.tie(NULL);

   string s = "HQ9";
   char c;
   while (cin >> c) {
      if (s.find(c) != string::npos) {
	 cout << "YES" << endl;
	 return 0;
      }
   }
   cout << "NO" << endl;
   


}

File: src/python/py-accepted/133A.py
#! python

# Problem #    : 133A
# Created on   : 2019-01-15 20:14:13


def Main():
    l = input()
    s = "HQ9"
    for ch in s:
        if (l.find(ch) != -1):
            print("YES")
            return
    print("NO")


if __name__ == '__main__':
    Main()
